app/controllers/course/experience_points_disbursement_controller.rb

```diff
@@ -22,7 +18,17 @@ def create # :nodoc:
 
   private
 
-  def experience_points_disbursement_params # :nodoc:
+  def load_and_authorize_experience_points_disbursement # :nodoc:
+    @experience_points_disbursement ||=
+      Course::ExperiencePointsDisbursement.new(current_course, send("#{params[:action]}_params"))
```

Comment:
OK, thanks. Will use `action_name`.

If I'm not mistaken, the current course is loaded already in the controller. Would it be redundant work to re-load it again using `course_id` in the model?